WordPress/Classes/ViewRelated/Likes/LikesListController.swift

```diff
@@ -148,22 +150,62 @@ class LikesListController: NSObject {
     ///   - success: Closure to be called when the fetch is successful.
     ///   - failure: Closure to be called when the fetch failed.
     private func fetchLikes(success: @escaping ([LikeUser], Int) -> Void, failure: @escaping (Error?) -> Void) {
+
+        var beforeStr = lastFetchedDate
+
+        if beforeStr != nil,
+           let modifiedDate = modifiedBeforeDate() {
+            // The endpoints expect a format like YYYY-MM-DD HH:MM:SS. It isn't expecting the T or Z, hence the replacingMatches calls.
+            beforeStr = ISO8601DateFormatter().string(from: modifiedDate).replacingMatches(of: "T", with: " ").replacingMatches(of: "Z", with: "")
+        }
+
         switch content {
         case .post(let postID):
             postService.getLikesFor(postID: postID,
                                     siteID: siteID,
-                                    before: lastFetchedDate,
+                                    before: beforeStr,
+                                    excludingIDs: excludeUserIDs,
                                     purgeExisting: isFirstLoad,
                                     success: success,
                                     failure: failure)
         case .comment(let commentID):
             commentService.getLikesFor(commentID: commentID,
                                        siteID: siteID,
-                                       before: lastFetchedDate,
+                                       before: beforeStr,
+                                       excludingIDs: excludeUserIDs,
                                        purgeExisting: isFirstLoad,
                                        success: success, failure: failure)
         }
     }
+
+    // There is a scenario where multiple users might like a post/comment at the same time,
+    // and then end up split between pages of results. So we'll track which users we've already
+    // fetched for the lastFetchedDate, and send those to the endpoints to filter out of the response
+    // so we don't get duplicates or gaps.
+    private func trackUsersToExclude() {
+        guard let modifiedDate = modifiedBeforeDate() else {
+            return
+        }
+
+        var fetchedUsers = [LikeUser]()
+        switch content {
+        case .post(let postID):
+            fetchedUsers = postService.likeUsersFor(postID: postID, siteID: siteID, after: modifiedDate)
+        case .comment(let commentID):
+            fetchedUsers = commentService.likeUsersFor(commentID: commentID, siteID: siteID, after: modifiedDate)
+        }
+
+        excludeUserIDs = fetchedUsers.map { NSNumber(value: $0.userID) }
+    }
+
+    private func modifiedBeforeDate() -> Date? {
+        guard let lastDate = likingUsers.last?.dateLiked else {
+            return nil
+        }
+
+        return Calendar.current.date(byAdding: .second, value: -1, to: lastDate)
```

Comment:
`value: -1,` 
I think we'd want this to be `value: 1,`. Dates are a little confusing because we're fetching in descending order. If 2:00:01 PM is the time of the last liker's 'dateLiked', and we want to fetch another page that includes any other likers who liked at 2:00:01 or before, we'd need to add a second to fetch all likers before 2:00:02.